css/src/components/icon.scss

```diff
@@ -0,0 +1,34 @@
+$icon-dimensions: 1.5rem !default;
+$icon-dimensions-small: 1rem !default;
+$icon-dimensions-medium: 2rem !default;
+$icon-dimensions-large: 3rem !default;
+
+.icon {
+	display: inline-flex;
+	align-items: center;
+	justify-content: center;
+	width: $icon-dimensions;
+	height: $icon-dimensions;
+
+	// Sizes
+
+	&.is-small {
```

Comment:
Would like to remove these if possible. Should the name be icon-small?

Reply:
they can be removed with `em` usage